### PR TITLE
Document enabling forward host headers for NGINX 0.25.0

### PR DESCRIPTION
**Pull request by catherineluse**
This PR addresses this issue https://github.com/rancher/docs/issues/1835

### CODE REVIEW

File: content/rancher/v2.x/en/installation/ha/helm-rancher/chart-options/_index.md [hunk 1]
```diff
@@ -156,6 +156,22 @@ You may terminate the SSL/TLS on a L7 load balancer external to the Rancher clus
 
 Your load balancer must support long lived websocket connections and will need to insert proxy headers so Rancher can route links correctly.
 
+#### Enabling Forward Host Headers
+
+_For Rancher v2.3.0+, which uses NGINX 0.25.0_
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/ha/helm-rancher/chart-options/_index.md]: Just reference for anyone using nginx 0.25.0.

Comment (catherineluse) [reply content/rancher/v2.x/en/installation/ha/helm-rancher/chart-options/_index.md]: updated

**Comment by deniseschannon** [comment]
@superseb Does this only apply to Rancher installs or would we need this in the RKE docs as well for anyone using the nginx ingress controller?

**Comment by superseb** [comment]
The reason we add it here is because it applies to a setup of Rancher we describe and it doesn't work in this newer version. I dont think we used to describe all changes in components that are used when setting up a cluster (or basically any component version besides Kubernetes), and all components provide changelogs, maybe we should link those in the docs.

**Pull request merged**